### PR TITLE
Increase lager truncation size to 15KB

### PR DESCRIPTION
**Pull request by balusu-murali**
- increase lager limit per message.

### CODE REVIEW

File: rebar.config [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {erl_opts, [warnings_as_errors,
             warn_export_all,
-            {lager_truncation_size, 8192}]}.
+            {lager_truncation_size, 15360}]}.   %% 15KB
```
Comment (vipin211) [inline rebar.config:3]: does this mean trucate after 15KiB in one log line?

Comment (balusu-murali) [reply rebar.config:3]: yeah.

**Review by vipin211 — APPROVED** [review]
Isn't it a problem if we are logging 15KiB of data in one log line?

**Comment by balusu-murali** [comment]
> Isn't it a problem if we are logging 15KiB of data in one log line?

our current limit is 8KB and sometime i dont see the full message.. so i increased it to 15KB.

**Review by NikolaBorisov — APPROVED** [review]

**Review by mdonohue — APPROVED** [review]

**Pull request merged**